docs/release/handbook.md

```diff
@@ -0,0 +1,288 @@
+
+### Manifests testing (1 week)
+
+The Manifests WG leads will be evaluating the final structure of manifests before Distribution owners start testing
+
+**Actions for other WGs:**
+- Provide updates and fixes for pending issues
+
+**Actions for Manifests WG:**
+- Ensure the manifests provide a deployable Kubeflow installation
+- Sync with other WGs on pending issues
+- After finished testing, create a new **vX.Y-rc.3** tag pointing at the commit of the **vX.Y-branch** release branch in case more bug fixes were cherry-picked
+
+**Success Criteria:** Manifests complete and fixes from working groups applied.
+
+### Documentation (3 weeks) (Starts the same time as Feature Freeze)
+
+Now that it’s been 3 weeks of fFeature freeze and continuous bug fixes we
+should all focus on ensuring that our docs are up to date, before distributions
+start testing. Features that are at risk or being worked up until the deadline
+can be documented last, ideally most of the feature work has landed already and
+those can start to be documented.
+
+**Actions for ALL WGs:** Help on keeping the docs up to date.
+
+**Actions for the Release Team:**
+- Request a list of features, from the Working Groups, that require updates to the documentation
+- Ensure the provided components versions match the documentation
+- Work alongside the Working Groups to bring the docs up to date
+
+**Success Criteria:** Documentation for this release completed. A new version
+in the website is cut.
+
+### Distribution testing (3 weeks)
+
+Distribution owners will start testing the latest RC tag and report back issues they find. Depending on the severity of the issues, and number of bug fixes that will accommodate them, the Manifests WG leads might need to cut a new RC tag.
+
+**Actions for other WGs:**
+
+- Evaluate which of the reported issues should be release blocking
+- Work on providing bug fixes for release blocking issues
+- Finalize their provided git revision. It should be stable (not RC) and include fixes for release blocking issues found during this time.
+
+**Actions for Manifests WG:**
+
+- \[OPTIONAL] Create a **vX.Y-rc.4** tag, depending on the number of created issues and bug fixes
+
+**Actions for Release Team:**
+
+- Official Blog post begins drafting and collating information from the Working Groups
+    - (Optional but encouraged) Working Groups start drafting WG-specific blog
+        posts, deep diving into their respective areas
+- Preparation for social media posts can start at the beginning of this phase
+- Release Manager: List the features, and ideally with docs, that made it into the release
+
+**Success Criteria:** Thumbs up from distribution representatives when their testing is complete.
+
+### Release
+
+We made it!
+
+**Actions for Manifests WG:**
+
+- Create the final **vX.Y** tag, pointing to the latest commit of the
+- Create an **vX.Y** tag in <https://github.com/kubeflow/kubeflow>
+- Cut a new GitHub release in <https://github.com/kubeflow/kubeflow>
```

Comment:
This should also be done on the manifests repository.

Also, this release should already exist on the kubeflow/kubeflow repository, since this is where the release team will need to get the release notes from the Notebooks working group from.

Reply:
pushed a commit